### ISSUE TITLE
Question: better and smaller button

### ISSUE THREAD

**Issue by Catzenjaeger** [Catzenjaeger/FigureLib_3#67]
i have add a button in last figure center panel and it works but it looks ....

have change this:
`echo openform('inputform', 'post', FUSION_REQUEST, array("class" => "",));
															echo "<div align='center'>\n";
															echo form_button("delete_from_collection", $locale['userfigure_002'], $locale['userfigure_002'], array("class" => "glyphicon glyphicon-minus"));
															echo "</div>\n";
														echo closeform();

to this:

`echo openform('inputform', 'post', FUSION_REQUEST, array("class" => "",));
															echo "<div align='center'>\n";
																	
															echo form_button("delete_from_collection", $locale[''], $locale[''], array("class" => "btn btn-danger btn-sx btn-round glyphicon glyphicon-minus"));
															
															echo "</div>\n";
														echo closeform();`

![unbenannt](https://cloud.githubusercontent.com/assets/19621858/17810895/1e7f7e20-6620-11e6-8640-c4bc529402a3.JPG)

it is possible to make it smaller or other?

**Comment by DavidGuetl** [comment]
Try btn-xs instead of btn-sx

**Comment by Catzenjaeger** [comment]
yup danke aber es ist verschoben da ich dort noch ein anderes symbol anzeigen lasse. der Platz ist groß genug aber es wird immer unten drunter angezeigt.

`						// COLUMN 8 (edit/add/remove)
						echo "<div class='col-lg-2 col-md-2 col-sm-2 col-xs-2'>\n";	
							
							if (iADMIN || iSUPERADMIN) {
							global $aidlink;
							$settings = fusion_get_settings();
								// ['cifg_0005'] = "Edit";
								
								echo "<a class='' href='".INFUSIONS."figurelib/admin.php".$aidlink."&amp;section=figurelib_form&amp;action=edit&amp;figure_id=".$_GET['figure_id']."'><span class='glyphicon glyphicon-edit' aria-hidden='true'></span></a>"; 
							}
								
	
							if (iMEMBER) {
									global $userdata;
														
										$resultuf = dbquery(
											"SELECT             
												fu.user_id, 
												fu.user_name, 
												fu.user_status, 
												fu.user_avatar, 
												fuf.figure_userfigures_figure_id,
												fuf.figure_userfigures_user_id          
										FROM ".DB_FIGURE_USERFIGURES." fuf
										INNER JOIN ".DB_USERS." fu ON fuf.figure_userfigures_user_id=fu.user_id  
										WHERE fuf.figure_userfigures_figure_id='".$data['figure_id']."'
										AND fu.user_id='".$userdata['user_id']."' 
										GROUP BY fu.user_id 
										");
															
											$rows = dbrows($resultuf);

										// USER HAVE THE FIGURE
										if ($rows > 0) { 
													
												while ($datauf = dbarray($resultuf)) {
																				
													if (isset($_POST['delete_from_collection'])) {
																
																dbquery("
																	DELETE FROM ".DB_FIGURE_USERFIGURES." 
																	WHERE figure_userfigures_figure_id=".$data['figure_id']." 
																	AND figure_userfigures_user_id=".$userdata['user_id']." 
																	");									
																redirect(clean_request("", array("delete_from_collection"), FALSE));

													}
												}
														echo openform('inputform', 'post', FUSION_REQUEST, array("class" => "",));
															//echo "<div align='center'>\n";
															//echo form_button("delete_from_collection", $locale['userfigure_002'], $locale['userfigure_002'], array("class" => "glyphicon glyphicon-minus"));
																	
															echo form_button("delete_from_collection", $locale[''], $locale[''], array("class" => "btn btn-danger btn-xs btn-round glyphicon glyphicon-minus"));
															
															/*echo form_button("delete_from_collection", $locale['minus'], $locale['minus'], 
															array(
															"class" => "btn btn-danger btn-sx btn-round",
															"span" => "glyphicon glyphicon-ok"
															));
															*/
															//echo "<button type='name='delete_from_collection' 'name='delete_from_collection' class='btn btn-success btn-sx btn-round'><span class='glyphicon glyphicon-ok'></span></button>";
															//echo "</div>\n";
														echo closeform();
														
										// USER DOSEN'T HAVE THE FIGURE					
										} else { 
																
													if (isset($_POST['add_to_collection'])) {
																		
														// Standard Values for Fields
														$inputArray = array(
														"figure_userfigures_figure_id" => $data['figure_id'], 
														"figure_userfigures_figure_id" => $userdata['user_id'], 
														);

														// SAVE DATA
														if (defender::safe()) {
																$inputArray = array(
																	"figure_userfigures_figure_id" => $data['figure_id'],
																	"figure_userfigures_user_id" => $userdata['user_id'],);
																dbquery_insert(DB_FIGURE_USERFIGURES, $inputArray, "save", array());	
																redirect(clean_request("", array("add_to_collection"), FALSE));									
														}
													}
												
														echo openform('inputform', 'post', FUSION_REQUEST, array("class" => "btn-sx",));
															//echo "<div align='center'>\n";
															//echo form_button("add_to_collection", $locale['userfigure_001'], $locale['userfigure_001'], array("class" => "btn-xs glyphicon glyphicon-plus"));
															//echo form_button("add_to_collection", $locale['plus'], $locale['plus'], array("class" => "btn btn-xs btn-success"));
															/*echo form_button("add_to_collection", $locale['plus'], $locale['plus'], 
															array(
															"class" => "btn btn-success btn-sx btn-round",
															"span" => "glyphicon glyphicon-ok"
															));	
															*/	

															echo form_button("add_to_collection", $locale[''], $locale[''], array("class" => "btn btn-success btn-xs btn-round glyphicon glyphicon-plus"));
															
															//echo "</div>\n";

														echo closeform();															
												}						
								}`

**Comment by Catzenjaeger** [comment]
looks better but i have breaking problem. The edit sign will show correct but the add/remove button will be shown in a new row and i dont know why..
